dss/error.py

```diff
@@ -81,7 +101,10 @@ def wrapper(*args, **kwargs):
             code = "read_only"
             title = "The DSS is currently read-only"
             stacktrace = ""
-            headers = {'Retry-After': 600}
+            headers = {'Retry-After': '600'}
+
+        if include_retry_after_header(return_code=status, method=request.method, uri=request.path):
```

Comment:
I implemented old testing that hit this, but you didn't want it to trigger errors inside of `dss/error.py` and I may have misunderstood, but thought you said to leave additional testing to a future ticket.

It will hit either `dss/error.py`, or `chalice/app.py` in different circumstances I believe.  Therefore, I'd strongly argue that we need both to be included as long as we have two error handling points (technically 3, since a 504 is it's own thing).

I'll try and make some fake endpoints in the test and see if that's adequate for testing.